client/src/pages/WeeklyTable.js

```diff
@@ -108,14 +111,20 @@ const WeeklyTable = ({
 	return (
 		<div>
 			<div>
+				<button className={"inner"} onClick={setPreviousMonday}>
+					Prev week
+				</button>
+				<a href="#" class="previous">
+					&laquo;
+				</a>
 				<button className={"inner"} onClick={setThisMonday}>
 					This week
 				</button>
-				<button className={"inner"} onClick={setPreviousMonday}>
-					Previous week
-				</button>
+				<a href="#" class="next">
+					&raquo;
+				</a>
 				<button className={"inner"} onClick={setNextMonday}>
```

Comment:
This button should be replaced by the arrow button. (or the content of this button should chnage to arrow)

And it should become a semantic ui &lt;Button&gt;